Gopkg.toml

```diff
@@ -24,11 +24,11 @@ required = [
 
 [[constraint]]
   name="sigs.k8s.io/controller-runtime"
-  version="v0.1.1"
+  version="v0.1.8"
```

Comment:
This is not needed, the current constraint does not have an operator which means `>=v0.1.1,<2.0.0`[0]

We are already at `v0.1.6` for controller-tools and `v0.1.7` for controller-runtime

[0]https://golang.github.io/dep/docs/Gopkg.toml.html

Reply:
That's a fair point. Given the breaking changes though I preferred to modify the versions and update afterwards. Any thoughts around that? I'm happy to change if needed.

Reply:
Well the line above reads `# STANZAS BELOW ARE GENERATED AND MAY BE WRITTEN - DO NOT MODIFY BELOW THIS LINE.` so IMHO we should just leave it as-is